app/views/transactions/_form_for_buy_or_sell.html.erb

```diff
@@ -25,8 +25,12 @@
 
       <div>
         <%= form.hidden_field :transaction_type, value: transaction_type, class: "form-control" %>
-        <%= form.submit transaction_type.capitalize, class: "p-4 cursor-pointer button button-primary" %>
+        <%= form.submit transaction_type.capitalize, class: "p-4 cursor-pointer button button-primary submit-button", data: { action: "click->transaction-form#handleSubmit" } %>
+        <div class="spinner hidden p-2">
+          <i class='fa fa-spinner animate-spin text-muted-foreground text-xl'></i>
+        </div>
       </div>  
     </div>
+    </div>
```

Comment:
This is causing an error, the tabs controller doesn't seem to recognize the 'panel' target.